android/java/org/chromium/chrome/browser/settings/BraveLeoPreferences.java

```diff
@@ -144,12 +148,4 @@ public void onResume() {
         setModel();
         checkLinkPurchase();
     }
-
-    private String getLinkURL() {
-        if (BraveQAPreferences.isLeoStagingUsed()) {
-            return LINK_SUBSCRIPTION_URL_STAGING;
```

Comment:
we can delete `LINK_SUBSCRIPTION_URL_STAGING` and `LINK_SUBSCRIPTION_URL`

Reply:
oh i am sorry i missed it.